README.md

```diff
@@ -10,37 +10,32 @@ HackerNews clone built with [Nuxt.js](https://nuxtjs.org).
   </a>
 </p>
 
-## Modes
+## Deploy
 
-- Universal: https://hn.nuxtjs.org
+- https://hn.nuxtjs.org
 
-> Hosted on [Now 2](https://zeit.co): `npm run build` + `now.json`
+> Hosted on [Vercel](https://vercel.com/): `npm run build`
```

Comment:
I guess the SPA mode is not that cool in Nuxt 3 anymore. Since we should many hosting services out-of-box, I am thinking maybe here we could deploy to all of them at the same time and provide the previews like:

- nuxt3-hn.vercel.app
- nuxt3-hn.netlify.app
- nuxt3-hn.cloudflare.app
- ....

Reply:
💡 Great idea.

Reply:
I like the idea. SPA is still cool since it shows that you simply need to disable one line but I am fine removing it for Nuxt 3 examples and instead showcase all the providers it supports.

Reply:
Nitro preset deploys are already deployed on (https://github.com/unjs/nitro-deploys). It is a huge double work for maintenance if we do it with each example.

Reply:
Oh, that makes sense then! Do you think we should keep Vercel only, or use the old format (Vercel for SSR and Netlify for SPA?)

Reply:
Yes we can keep using these two, forgot about this double maintenance